addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3363,7 +3361,7 @@ var SnippetsMenu = Widget.extend({
                         self.options.wysiwyg.odooEditor.observerActive('dragAndDropCreateSnippet');
 
 
-                        _.defer(async function () {
+                        setTimeout(async () => {
```

Comment:
never use setTimeout directly please, instead you should import `browser` from `@web/core/something` and use `browser.setTimeout`

Reply:
why ?

Reply:
yes i see